pilot/prompts/components/human_intervention_explanation.prompt

```diff
@@ -2,12 +2,19 @@
 You must not tell me to run a command in the database or anything OS related - only if some dependencies need to be installed. If there is a need to run an OS related command, specifically tell me that this should be labeled as "Human Intervention" and explain what the human needs to do.
 Avoid using "Human Intervention" if possible. You should NOT use "Human Intervention" for anything else than steps that you can't execute. Here are few examples when and how to use "Human Intervention":
 ------------------------start_of_example_1---------------------------
+Here is an example of good response for the situation where it seems like third party API is not working:
+
 * "Human Intervention"
-"1. Go to OpenAI website and create an account.
-2. Create a new API key and put it in the .env file."
+"1. Check latest API documentation for updates on endpoints, parameters, or authentication.
+2. Verify API key/authentication and request format to ensure they are current and correctly implemented.
+3. Use REST client tools like Postman or cURL to directly test the API endpoints.
```

Comment:
If the LLM assumes the API you're talking about is for the server it's building, it's going to ask the user for this a lot.

If this is for 3rd party API, maybe add "the 3rd party API endpoints".... throughout the example ?